CONTRIBUTING.md

```diff
@@ -2,71 +2,20 @@
 
 We're excited you're considering contributing to our design system. If you have a question, noticed a bug, or have suggestions, then please submit an issue or create a pull request.
 
-One of our goals is to ensure a welcoming environment for all contributors. Please take a look at our [Code of Conduct](CODE_OF_CONDUCT.md) to learn more.
+One of our goals is to ensure a welcoming environment for all contributors. 
+Please take a look at our [Code of Conduct](CODE_OF_CONDUCT.md) to learn more.
+If you are interessted in running this site locally please take a look at [setting up your local development environment](/README.md#running-locally). 
```

Comment:
I think it'd be good to add a paragraph break here. The sentence about running the site locally feels like a shift from the CoC thought. What do you think about another heading like **Getting started** that could encompass the local dev environment and wiki points?

Interested has an extra "s".